tests/integration/bulk_upsert_simple_it/bulk_upsert.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <ydb-cpp-sdk/client/driver/driver.h>
+#include <ydb-cpp-sdk/client/table/table.h>
+
+using namespace NYdb;
+using namespace NYdb::NTable;
+
+struct TRunArgs {
+    TDriver driver;
+    std::string path;
+};
+
+struct TLogMessage {
+    uint64_t pk;
```

Comment:
Неправильный кодстайл
```suggestion
    uint64_t Pk;
```